AppConnectSwift/LoginViewController.swift

```diff
@@ -10,7 +10,9 @@ class LoginViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        
+        //TODO: remove these when done debugging
+        usernameField.text = "sub001@sqa.com"
+        passwordField.text = "Password1"
```

Comment:
These credentials should be removed (and will have to be expired). In general we shouldn't expose any credentials, however harmless.